chapter2/tests/t20/terraform/base/infra.auto.tfvars

```diff
@@ -0,0 +1,74 @@
+client  = "maxim-omelchenko"
+project = "wp"
+
+tags = {
+  project    = "wp"
+  terraform  = true
+  git        = "https://github.com/rtest12/devops-camp-2023.git"
+  branch     = "Task_20"
+  created_by = "maxim-omelchenko"
+  created_at = "06/12/2023"
+  updated_at = "06/12/2023"
+}
+
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ fqdn vars                                                                                                        │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+root_domain         = "saritasa-camps.link"
+site_url            = "tf-maxim-omelchenko.saritasa-camps.link"
```

Comment:
Let's move this to `locals`, so you would combine `root_domain` and `subdomain` there?

Reply:
> Let's move this to `locals`, so you would combine `root_domain` and `subdomain` there?

@kseniyashaydurova 
Done